README.md

```diff
@@ -95,16 +95,7 @@ Here is what ColorPaletteOverride.xaml would contain if you want both light and
 ```csharp
 		protected override void OnLaunched(LaunchActivatedEventArgs e)
 		{
-			// Set a default palette to make sure all colors used by MaterialResources exist
-			this.Resources.MergedDictionaries.Add(new Material.MaterialColorPalette());
-			
-			// Overlap the default colors with the application's colors palette. 
-			// TODO: Replace ms-appx:///Views/ColorPaletteOverride.xaml with your resourceDictionary.
-			this.Resources.MergedDictionaries.Add(new ResourceDictionary() { Source = new Uri("ms-appx:///Views/ColorPaletteOverride.xaml") });
-			
-			// Add all the material resources. Those resources depend on the colors above, which is why this one must be added last.
-			this.Resources.MergedDictionaries.Add(new Material.MaterialResources());
-			
```

Comment:
We don't need to do that anymore with latest Uno ?

Reply:
like the lastest uno dev? or already available in the stable version?

Reply:
I don't understand what updating Uno changes for this? it's still required to do it in 3 steps.

Reply:
Ahh yes. Uno.Material.Resource.Init was added previously but it caused weird bugs so it was reverted.

Reply:
Ho! seems to work well for me until now with the latest uno stable. should I revert?